.github/workflows/build.yml

```diff
@@ -55,18 +55,14 @@ jobs:
           submodules: recursive
       - name: Build site and check links
         run: make build
+      - name: Move docs
+        run: make move-docs
```

Comment:
@chenglu @AlexV525 I'm not 100% sure as I can't see what happens in github.com/chenglu/site-flutter.cn and I don't know this site's infrastructure well, but I think this PR may be the source of the issue where active entries aren't being highlighted correctly in the sidenav.

Sidenav generation uses the permalinks from `sidenav.yml` and each page's `page.url` to see if it matches a sidenav entry. This PR changes it so `make move-docs` is ran after the final `make build`, so the pages under `/docs` have an incorrect `page.url` when building the site and sidenav. 

I think moving `make move-docs` before the `make build` would fix this, but let me know if it doesn't or I'm missing something. Thanks!

Reply:
Thanks for the investigation! Submitted https://github.com/cfug/flutter.cn/pull/1261